app/scripts/views/base.js

```diff
@@ -55,7 +55,9 @@ function (Cocktail, _, Backbone, Raven, $, p, AuthErrors,
 
     err = this._normalizeError(err);
 
-    this.logError(err);
+    if (shouldLog) {
+      this.logError(err);
```

Comment:
Something fishy is happening here, on L490, [logError checks if the error has been logged](https://github.com/mozilla/fxa-content-server/blob/issue-2519-ephemeral-logged-twice/app/scripts/views/base.js#L490), and if so, does not re-log. That was supposed to fix this issue.